### ISSUE TITLE
Ability to star content

### ISSUE THREAD

**Issue by jack-white-securecheck** [Bartek532/syncreads#87]
### Feature description

I think it could be great if you can have the ability to star content that you added. For me, I'm not much of a reader, but as a software engineer, I have many different pieces of documentation that read frequently, so to be able to star them I can have the ability to quickly access them when I'm looking for a specific article.

With that, then perhaps there could be a "Starred" section on the Home page in the Dashboard, where I can instantly see all the pieces of content that I've starred. 

In the Library, there could then also be a "Star" tab, where I can view all starred contents.

### Additional Context

Additional details here...

**Comment by Bartek532** [comment]
Nice idea! But don't you think that introducing search will fix this issue for you?

Of course, displaying starred content as a separate content could be useful, but I wonder what should I take higher priority 🤔

**Comment by jack-white-securecheck** [comment]
Hmm I see what you mean, especially if you already have a plan to introduce 'Tags' (Issue: #88), then I'd say having a search feature might take priority. 

For the search functionality, would you only be able to search by the content's Title, or the actual content as well? I suppose you can always refine it as you go along.

Not sure if you worked with it before, but [Algolia](https://www.algolia.com/) might be a good starting point.

**Comment by Bartek532** [comment]
Yes, definitely doing full-text search will be super handy here.

**Comment by jack-white-securecheck** [comment]
Nice one! Yeah that would be a good feature!
